ui/app/adapters/job.js

```diff
@@ -255,7 +255,12 @@ export default class JobAdapter extends WatchableNamespaceIDs {
   urlForQuery(query, modelName, method) {
     let baseUrl = `/${this.namespace}/jobs/statuses`;
     if (method === 'POST' && query.index) {
-      return `${baseUrl}?index=${query.index}`;
+      baseUrl += baseUrl.includes('?') ? '&' : '?';
+      baseUrl += `index=${query.index}`;
+    }
+    if (method === 'POST' && query.jobs) {
+      baseUrl += baseUrl.includes('?') ? '&' : '?';
+      baseUrl += 'namespace=*';
```

Comment:
Implementation detail: even though I'm posting jobs with {id:..., namespace:...} in the body, if I don't include the namespaces I'm looking for in my queryParams, I won't get the requested jobs back.

This had the effect of them appearing to have been GC'd to the web user (they showed up on my GET, but not on my subsequent POST).